packages/react-app/src/views/BuilderListView.jsx

```diff
@@ -80,6 +83,32 @@ const BuilderStatusCell = ({ status }) => {
   );
 };
 
+const secondsPerDay = 24 * 60 * 60;
+const BuilderStreamCell = ({ stream }) => {
+  if (!stream) return <Box>-</Box>;
+
+  const cap = ethers.utils.parseUnits(stream.cap);
+  const frequency = stream.frequency;
+  const last = stream.lastContract;
+  const frequencyDays = frequency / secondsPerDay;
+  const vestedPercentage = (new Date().getTime() / 1000 - last) / frequency;
+  const vestedAmount = cap.mul(Math.round(new Date().getTime() / 1000 - last)).div(frequency);
+  const available = cap.lt(vestedAmount) ? cap : vestedAmount;
+
+  const capStr = ethers.utils.formatEther(cap);
+  const availableStr = ethers.utils.formatEther(available);
+  return (
+    <Flex align="center" justify="center" direction="column" px={4} mt={4}>
+      <Box mb={1}>
+        Ξ {parseFloat(availableStr).toFixed(4)} / {parseFloat(capStr).toFixed(1)} @ {frequencyDays}d
```

Comment:
Make sure this doesn't break the line

![image](https://user-images.githubusercontent.com/2486142/157725391-86b475b8-b805-4925-9b54-2a258bd0d25e.png)

Reply:
I've added a nowrap on the box, but doing this causes the timestamp in the last column to wrap. I'll leave it with the nowrap, but we might want to consider a better solution for smaller screens.